Verslag/methodology.tex

```diff
@@ -1,32 +1,71 @@
+For the research it was decided that all the selected applications had come from the site Koodous and were confirmed to be malicious with the site VirusTotal.
+This had been done individually by all team members to find applicable APK files for the research.
+Some checks were performed manually to see if the applications contained any pointers that indicated that they contained malware.
+The application was checked on its availability for download on the Google Play store.
+The potentially malicious application was compared with the variant from the Google Play store.
+A few examples of these pointers were differing sizes or if both had the exact same package name as the Google Play store variant while differing in permissions or certificates.
+If it had one of those pointers, then it likely was an indicator that there was something extra hidden in the code.
 
-The application was retrieved from both Koodous and the Google Play Store. The APK from Google Play was downloaded using the apkcombo.com tool. Both APK files were uploaded to VirusTotal to check for any differences in permissions between the two versions of the application. Anything VirusTotal had marked anything as malicious, was noted. This was up to the discretion of every individual team member to decide if they decided to work on the chosen application for the duration of the project.
+The application was retrieved from both Koodous and the Google Play Store.
+The APK from Google Play was downloaded using the apkcombo.com tool.
+Both APK files were uploaded to VirusTotal to check for any differences in permissions between the two versions of the application.
+Anything VirusTotal had marked anything as malicious, was noted.
+This was up to the discretion of every individual team member to decide if they decided to work on the chosen application for the duration of the project.
 
 \subsection{Behavior analysis}
 
-The research required that five applications were examined. This had been done on a virtual computer with android studio installed or locally. During the installation of the application the network traffic had been captured. This had been done using Wireshark and mitmproxy. The network traffic was not relevant for behavior analysis but was captured as part of the network analysis.
+The research required that each team member has one application to examine.
+This had been done on a virtual computer with android studio installed or locally.
```

Comment:
This had been done on a virtual machine, with android studio installed, or locally.